src/circleci/rollcage/core.clj

```diff
@@ -255,13 +255,15 @@
           {:params {:thread-name (.getName thread)}}))
 
 (defn setup-uncaught-exception-handler
-  "Setup handler to report all uncaught exceptions
-  to rollbar."
-  [client]
-  (Thread/setDefaultUncaughtExceptionHandler
+  "Setup handler to report all uncaught exceptions to rollbar, and optionally
+  to an additional handler."
+  ([client] (setup-uncaught-exception-handler client identity))
+  ([client handler]
+   (Thread/setDefaultUncaughtExceptionHandler
     (reify Thread$UncaughtExceptionHandler
       (uncaughtException [_ thread ex]
-        (report-uncaught-exception "critical" client ex thread)))))
+        (report-uncaught-exception "critical" client ex thread)
+        (handler ex))))))
```

Comment:
`handler` needs to have the signature `[thread ex]` to be generally useful.

Reply:
Good call; thanks.